source/connection-troubleshooting.txt

```diff
@@ -4,194 +4,184 @@
 Connection Troubleshooting
 ==========================
 
-.. default-domain:: mongodb
-
 .. contents:: On this page
    :local:
    :backlinks: none
    :depth: 2
    :class: singlecol
 
-This page offers potential solutions to issues you might see when
-connecting to a MongoDB instance or replica set while using the
-{+driver-long+}.
+.. facet::
+   :name: genre
+   :values: reference
 
-.. note::
+.. meta::
+  :keywords: code example, disconnected, deployment
```

Comment:
Not sure if this has an impact on the actual functionality of the meta tag, but it loks like there's a space missing here
```suggestion
.. meta::
   :keywords: code example, disconnected, deployment
```